ratings.php

```diff
@@ -87,100 +94,125 @@
 /**
  * Display all the players' ratings.
  *
- * @param int    $start      Starting offset for page navigator
- * @param string $msg        Message to display (if non-empty string)
- * @param string $msg_class  CSS class for displaying message text
+ * @param int    $start     Starting offset for page navigator
+ * @param string $msg       Message to display (if non-empty string)
+ * @param string $msg_class CSS class for displaying message text
  */
 function chess_ratings($start = 0, $msg = '', $msg_class = 'errorMsg')
 {
-	global $xoopsDB, $xoopsUser, $xoopsModule, $xoopsTpl;
+    global $xoopsDB, $xoopsUser, $xoopsModule, $xoopsTpl;
+
+    // Display ratings.
+
+    // Get maximum number of items to display on a page, and constrain it to a reasonable value.
+
+    $max_items_to_display = chess_moduleConfig('max_items');
+
+    $max_items_to_display = min(max($max_items_to_display, 1), 1000);
 
-// Display ratings.
+    $games_table = $xoopsDB->prefix('chess_games');
 
-	// Get maximum number of items to display on a page, and constrain it to a reasonable value.
-	$max_items_to_display = chess_moduleConfig('max_items');
-	$max_items_to_display = min(max($max_items_to_display, 1), 1000);
+    $ratings_table = $xoopsDB->prefix('chess_ratings');
 
-	$games_table   = $xoopsDB->prefix('chess_games');
-	$ratings_table = $xoopsDB->prefix('chess_ratings');
-	
-	// Two queries are performed, one without a limit clause to count the total number of rows for the page navigator,
-	// and one with a limit clause to get the data for display on the current page.
-	// SQL_CALC_FOUND_ROWS and FOUND_ROWS(), available in MySQL 4.0.0, provide a more efficient way of doing this.
+    // Two queries are performed, one without a limit clause to count the total number of rows for the page navigator,
 
-	$result = $xoopsDB->query("
+    // and one with a limit clause to get the data for display on the current page.
+
+    // SQL_CALC_FOUND_ROWS and FOUND_ROWS(), available in MySQL 4.0.0, provide a more efficient way of doing this.
+
+    $result = $xoopsDB->query(
+        "
 		SELECT    COUNT(*)
 		FROM      $ratings_table AS p
-	");
-	list($num_items) = $xoopsDB->fetchRow($result);
-	$xoopsDB->freeRecordSet($result);
+	"
+    );
+
+    [$num_items] = $xoopsDB->fetchRow($result);
```

Comment:
Why was this changed into an array? It's used when instantiating \XoopsPageNav() and it expects an integer, not an array as a parameter.

Reply:
It was originally:
`list($num_items) = $xoopsDB->fetchRow($result);`

So it was converted to shorthand:
https://stitcher.io/blog/array-destructuring-with-list-in-php